src/SFA.DAS.AssessorService.Application/Handlers/OrganisationStandards/OrganisationStandardVersionOptInHandler.cs

```diff
@@ -61,12 +61,20 @@ public async Task<OrganisationStandardVersion> Handle(OrganisationStandardVersio
                     Comments = request.Comments,
                     Status = request.Status
                 };
+                
+                if (request.OptInFollowingWithdrawal)
+                {
+                    await _repository.UpdateOrganisationStandardVersion(entity);
```

Comment:
Do you still need the existing version check in this branch of the code as well as a validation check, or are we happy that if it doesn't exist, then the update will just do nothing?

Reply:
Added existing version check in for completeness